src/python/circllhist/wrapper.py

```diff
@@ -169,3 +177,56 @@ def compress_mbe(self, mbe):
         - return the compressed histogram as new value
         """
         return Circllhist(ffi.C.hist_compress_mbe(self._h, mbe), gc=True)
+
+    def plot(self, **kwargs):
+        """
+        Plot histogram using matplotlib.
+        Depends on matplotlib being available.
+        Retruns an axes object.
+
+        Hint: Use H.plot().set_xlim(x_min, x_max) to adjust value range.
+
+        mbe-compression is applied before plotting so squash bins which are deemed to small.
```

Comment:
"mbe-compression is applied before plotting **to** squash bins which are deemed **too** small." ?